src/button/notification.scss

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-button-notification($status: primary) {
```

Comment:
What if we apply this class not on a button but on an icon?

Reply:
Hmm… that might be possible.  I'll investigate later this week if I have time before I leave (I'm off Thurs-Fri).